### PR TITLE
Add histograms for turker missions, labels, and time spent on admin page

### PR DESCRIPTION
**Pull request by tongning**
This PR adds histograms for turker-related metrics to the admin page - specifically missions, labels, and time spent.

Before:
hist-before
After:
turker-grph

### CODE REVIEW

File: public/javascripts/Admin/src/Admin.js [hunk 1]
```diff
@@ -768,44 +768,66 @@ function Admin(_, $, c3, turf, difficultRegionIds) {
...
+                          var regChart = getVegaLiteHistogram(regTimes, regStats.mean, regStats.median, regHistOpts);
+                          var turkerChart = getVegaLiteHistogram(turkerTimes, turkerStats.mean, turkerStats.median, turkerHistOpts);
+                          var anonChart = getVegaLiteHistogram(anonTimes, anonStats.mean, anonStats.median, anonHistOpts);
+
+                          $("#all-audittimes-std").html((allStats.std).toFixed(2) + " Minutes");
+                          $("#reg-audittimes-std").html((regStats.std).toFixed(2) + " Minutes");
+                          $("#turker-audittimes-std").html((turkerStats.std).toFixed(2) + " Minutes");
+                          $("#anon-audittimes-std").html((anonStats.std).toFixed(2) + " Minutes");
+
+                          var combinedChart = {"hconcat": [allChart, regChart, turkerChart, anonChart]};
```
Comment (misaugstad) [inline public/javascripts/Admin/src/Admin.js]: Very minor, but I would just go with all, turker, registered, anon; seems weird to split up registered and anon users for some reason.

File: public/javascripts/Admin/src/Admin.js [hunk 2]
```diff
@@ -1086,113 +1108,201 @@ function Admin(_, $, c3, turf, difficultRegionIds) {
...
+                        var turkerChart = getVegaLiteHistogram(turkerData[0], turkerStats.mean, turkerStats.median, turkerHistOpts);
+                        var turkerFilteredChart = getVegaLiteHistogram(turkerData[0], turkerFilteredStats.mean, turkerFilteredStats.median, turkerFilteredHistOpts);
+                        var anonChart = getVegaLiteHistogram(anonData[0], anonStats.mean, anonStats.median, anonHistOpts);
+
+                        $("#missions-std").html((allStats.std).toFixed(2) + " Missions");
+                        $("#reg-missions-std").html((regStats.std).toFixed(2) + " Missions");
+                        $("#turker-missions-std").html((turkerStats.std).toFixed(2) + " Missions");
+                        $("#anon-missions-std").html((anonStats.std).toFixed(2) + " Missions");
+
+                        var combinedChart = {"hconcat": [allChart, regChart, turkerChart, anonChart]};
```
Comment (misaugstad) [inline public/javascripts/Admin/src/Admin.js]: Same here with ordering

File: public/javascripts/Admin/src/Admin.js [hunk 3]
```diff
@@ -1086,113 +1108,201 @@ function Admin(_, $, c3, turf, difficultRegionIds) {
...
+                        var turkerChart = getVegaLiteHistogram(turkerData[0], turkerStats.mean, turkerStats.median, turkerHistOpts);
+                        var turkerFilteredChart = getVegaLiteHistogram(turkerData[0], turkerFilteredStats.mean, turkerFilteredStats.median, turkerFilteredHistOpts);
+                        var anonChart = getVegaLiteHistogram(anonData[0], anonStats.mean, anonStats.median, anonHistOpts);
+
+                        $("#all-labels-std").html((allStats.std).toFixed(2) + " Labels");
+                        $("#reg-labels-std").html((regStats.std).toFixed(2) + " Labels");
+                        $("#turker-labels-std").html((turkerStats.std).toFixed(2) + " Labels");
+                        $("#anon-labels-std").html((anonStats.std).toFixed(2) + " Labels");
+
+                        var combinedChart = {"hconcat": [allChart, regChart, turkerChart, anonChart]};
```
Comment (misaugstad) [inline public/javascripts/Admin/src/Admin.js]: And here

**Comment by jonfroehlich** [comment]
Would be great to get this deployed onto production... @manaswis?

**Comment by misaugstad** [comment]
@jonfroehlich @manaswis I want to test this PR this week. There is another PR of mine, #1156 that I need to modify based on the changes from this PR, so I am planning on doing all that right in a row later this week.

**Comment by manaswis** [comment]
@misaugstad Okay. So the plan is for you to test this PR and merge it and work on #PR 1156 and then I will deploy both of them together?

**Comment by misaugstad** [comment]
@manaswis yep

**Pull request merged**